src/carla_cyber_bridge/static.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+
+#
+# Copyright (c) 2018-2019 Intel Corporation
+#
+# This work is licensed under the terms of the MIT license.
+# For a copy, see <https://opensource.org/licenses/MIT>.
+#
+"""
+Classes to handle Carla statics
+"""
+
+from carla_cyber_bridge.traffic_participant import TrafficParticipant
+
+from cyber.carla_bridge.carla_proto.proto.carla_marker_pb2 import ColorRGBA
+from modules.perception.proto.perception_obstacle_pb2 import PerceptionObstacle
+
+
+class Static(TrafficParticipant):
+
+    """
+    Actor implementation details for vehicles
+    """
+
+    def __init__(self, uid, name, parent, node, carla_actor):
+        """
+        Constructor
+
+        :param uid: unique identifier for this object
+        :type uid: int
+        :param name: name identiying this object
+        :type name: string
+        :param parent: the parent of this
+        :type parent: carla_cyber_bridge.Parent
+        :param node: node-handle
+        :type node: carla_cyber_bridge.CarlaCyberBridge
+        :param carla_actor: carla vehicle actor object
```

Comment:
forgot to update the comment when copied over from vehicle.py?

Reply:
Sorry forgot to update the comment... I have updated the static.py. :)